packages/runner/index.d.ts

```diff
@@ -8,3 +8,4 @@
 
 /// <reference path="../ts/index.d.ts" />
 /// <reference path="../driver/types/internal-types.d.ts" />
+/// <reference path="../driver/types/spec-types.d.ts" />
```

Comment:
```suggestion
/// <reference path="../driver/types/spec-types.d.ts" />

```

Reply:
updated in [c264f20](https://github.com/cypress-io/cypress/pull/20949/commits/c264f206d6ffcdfec536f4e2800f20b9a59bfcf6)